networkit/cpp/centrality/ApproxCloseness.cpp

```diff
@@ -14,10 +14,10 @@
 
 namespace NetworKit {
 
-typedef struct ListEntry_struct {
+using ListEntry = struct ListEntry_struct {
     node node_val;
     edgeweight dist_val;
-} ListEntry;
+};
```

Comment:
Just get rid of `ListEntry_struct` here. (In C, struct names and type names are not necessarily identical but in C++ that is indeed the case.)